packages/lib/src/ui/components/Image/Image.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import styled from "@emotion/styled"
+import NextImage from "next/image"
+import { Image as ImageType } from "../../../../src/types/data"
+
+interface ImageProps {
+  image: ImageType
+  className?: string
+}
+
+export const Image: React.FC<ImageProps> = (props) => (
```

Comment:
Zvlášť u takhle jednoduchých komponent je dobré props destrukturovat:
```suggestion
export const Image: React.FC<ImageProps> = ({className, image, }) => (
```